app/client/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,42 @@
+import { Grid, Button } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        display: 'flex',
+        padding: theme.spacing(1),
+        background: '#DDDDDD',
+        borderRadius: '7px',
+        gap: theme.spacing(1)
+    },
+    button: {
+        width: '100%',
+        borderRadius: '7px',
+    }
+}));
+
+interface Props {
+    value: number
```

Comment:
Value is really generic. Please add a comment explaining what this value is or represents.  I'm pretty sure it's tab index.  Renaming this would also make things clearer.